### ISSUE TITLE
New Tab Created Every Time Stencil Is Started

### ISSUE THREAD

**Issue by GLoganDR** [ionic-team/stencil#2468]
**Stencil version:**

```
 @stencil/core@1.12.7 
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
Every time a user (dev) runs npm start, Stencil creates a new tab. This was replicated with two Ionic devs on a call during a demo. The browser being used is Chrome.

**Expected behavior:**
Stencil should detect a current tab running and refresh that instead of opening a new Chrome tab every time the npm start command is executed.

**Steps to reproduce:**
Run npm start in the console. Go back to the console, CTRL + c. Run npm start again in the console. Return to the Chrome page, there should be more than one tab of the same thing. (I'm using Stencil for Web Components, so there are two running instances of that in my browser).

**Related code:**

```tsx
// insert any relevant code here
```

**Other information:**
I was on a call with two Ionic engineers when I was able to demonstrate this behavior in real-time. They suggested I create an Issue here.

**Comment by tricki** [comment]
Not a fix, but you could run `npm start --no-open` to prevent a tab being opened. But I agree, it would be nice if Stencil automatically detected the already open browser.

**Comment by GLoganDR** [comment]
> Not a fix, but you could run `npm start --no-open` to prevent a tab being opened. But I agree, it would be nice if Stencil automatically detected the already open browser.

You’re not wrong, but I’m not sure that is a usable solution, because then you have to use two separate start scripts to make sure the initial tab opens the first time. I added some things in previous apps to stop this behavior, but I’m not sure Stencil/Ionic would want to do it the way I did it.

**Comment by rwaskiewicz** [comment]
👋 

I apologize this wasn't responded to (especially after a call with folks from Ionic who probably should have been on the look out for this). This seems like a reasonable request to me. I've labeled it to get it ingested into our backlog, as well as for the community to upvote.
